bayes_opt/bayesian_optimization.py

```diff
@@ -101,8 +110,26 @@ class BayesianOptimization(Observable):
     set_bounds()
         Allows changing the lower and upper searching bounds
     """
+
     def __init__(self, f, pbounds, random_state=None, verbose=2,
-                 bounds_transformer=None):
+                 bounds_transformer=None,
+                 dataset_path=None, output_path=None, target_column=None):
```

Comment:
`output_path` is not documented in the class docstring

Reply:
fixed